test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/JSExportStaticTest.scala

```diff
@@ -0,0 +1,185 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2017, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.jsinterop
+
+import scala.scalajs.js
+import js.annotation._
+
+import org.scalajs.testsuite.utils.AssertThrows._
+import org.scalajs.testsuite.utils.Platform._
+
+import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.Test
+
+class JSExportStaticTest {
+  import JSExportStaticTest._
```

Comment:
Consider not importing this here, for better clarity in the nested tests (not entirely sure it's more readable though).